tests/trainer/logging_/test_eval_loop_logging.py

```diff
@@ -676,32 +675,6 @@ def val_dataloader(self):
     trainer.fit(model)
 
 
-@pytest.mark.parametrize(
-    ["kwargs", "expected"],
-    [
```

Comment:
These cases should still be tested

Reply:
have updated the current test with these cases.